deepchecks/recommender/checks/user_session_length_drift.py

```diff
@@ -0,0 +1,139 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing the popularity bias check."""
+import typing as t
+
+import pandas as pd
+
+from deepchecks.core import CheckResult
+from deepchecks.core.condition import ConditionCategory, ConditionResult
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.recommender import Context
+from deepchecks.tabular import TrainTestCheck
+from deepchecks.utils.distribution.drift import calc_drift_and_plot
+from deepchecks.utils.validation import is_sequence_not_str
+
+__all__ = ['UserSessionDrift']
+
+class UserSessionDrift(TrainTestCheck):
+    """
+    
+    Check for user session length bias in train and validation sets. This check compares the distribution of user session lengths between the train and validation sets to identify potential bias or discrepancies. It helps assess whether the two sets exhibit similar patterns in terms of user session lengths or if there are significant differences that could introduce bias in the recommender system.
```

Comment:
drift, not bias. Should use a clearer way than bias, which most people will understand as relating to something "unfair".

Reply:
Fixed. 
6f27711b7ee9fe8a30a16ae9e60328df58674c55